src/screen/dashboard.rs

```diff
@@ -17,6 +17,7 @@ use crate::widget::{selectable_text, Element};
 const SAVE_AFTER: Duration = Duration::from_secs(3);
 
 pub struct Dashboard {
+    config: dashboard::Config,
```

Comment:
Since we don't plan to ever modify config at runtime, it can be a presentation only item. We can just pass it down during `view` from our root state and reference it when building the view tree.

Reply:
I see what you mean, how does it look now?